lms/envs/common.py

```diff
@@ -4306,7 +4306,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # E-Commerce Commerce Coordinator Configuration
 COMMERCE_COORDINATOR_URL_ROOT = 'http://localhost:8140'
 COMMERCE_COORDINATOR_REFUND_PATH = '/lms/refund/'
-COMMERCE_COORDINATOR_REFUND_SOURCE_SYSTEMS = ('SET-ME-PLEASE',)
+COMMERCE_COORDINATOR_USER_RETIREMENT_PATH = '/lms/user_retirement/'
+COMMERCE_COORDINATOR_REFUND_SOURCE_SYSTEMS = ('commercetools',)
```

Comment:
Is this something that should live in the Open Source common config? It feels like it might be more appropriate to be set in our (2U) internal config files.

Reply:
Yes! Sorry, I had set it for local testing, will change it back to the 'SET-ME' value.